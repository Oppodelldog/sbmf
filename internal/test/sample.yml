attributes:
  version: 0
messages:
  Coordinate:
  - X: i32
  - Y: i32
