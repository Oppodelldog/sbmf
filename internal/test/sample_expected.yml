attributes:
  version: 1
messages:
  Coordinate:
  - X: i32
  - Y: i32
